eth/client.go

```diff
@@ -524,7 +524,7 @@ func (c *Client) SubscribeToJobEvent(ctx context.Context, logsCh chan types.Log)
 
 	q := ethereum.FilterQuery{
 		Addresses: []common.Address{c.jobsManagerAddr},
-		Topics:    [][]common.Hash{[]common.Hash{abiJSON.Events["NewJob"].Id()}, []common.Hash{common.BytesToHash(common.LeftPadBytes(c.account.Address[:], 32))}},
+		Topics:    [][]common.Hash{[]common.Hash{abiJSON.Events["NewJob"].Id()}, []common.Hash{common.BytesToHash(common.LeftPadBytes(transcoderAddr[:], 32))}, []common.Hash{common.BytesToHash(common.LeftPadBytes(broadcasterAddr[:], 32))}},
```

Comment:
According to these comments: https://github.com/ethereum/go-ethereum/blob/019dca9ba2f0ad91047129b95d91b95a9e71650e/interfaces.go an empty slice will match any topic. Looks like in `livepeer.go` we pass in an empty address if we don't care about the transcoder or broadcaster address for a job event, but `common.BytesToHash(common.LeftPadBytes(...))` might lead to a hash with all zeros? Not sure about this, but probably worth testing out and if that is what happens we can just explicitly pass in an empty slice for a topic we don't care about

Reply:
I tested it out with a local geth node - seems to work